docs/harvest_postgresql_tutorial.md

```diff
@@ -0,0 +1,63 @@
+# Harvest with PostgreSQL
+
+## Installation and Setup:
+
+### Harvest
+
+```bash
+pip install -e 'git+https://github.com/tfukaza/harvest.git#egg=harvest'
+```
+
+### SQLAlchemy
+
+```bash
+pip install SQLAlchemy
+pip install pg8000 # Connects SQLAlchemy to the PostgreSQL database
+```
+
+### PostgreSQL
+
+See PostgreSQL installation guide [here](https://www.postgresql.org/download/)
+
+Now create a user with a password and give it access to the default postgres database.
+
+```bash
+sudo -u postgres -i
+psql 
+CREATE SCHEMA harvest;
+CREATE USER username PASSWORD 'password';
+GRANT ALL ON SCHEMA harvest TO username;
+GRANT ALL ON ALL TABLES IN SCHEMA harvest TO username;
+GRANT ALL PRIVILEGES ON DATABASE postgres TO username;
+\q
+```
+
+## Implementation:
+
+```python
+from harvest.storage.database_storage import DBStorage
+from harvest.algo import BaseAlgo
+from harvest.trader import Trader
+
+# The formatting here is dialect+driver://username:password@host:port/database
+postgre_storage = DBStorage('postgresql+pg8000://username:password@localhost/postgres')
```

Comment:
The user will also have to pass in the dialect, driver, host, and db name and there are certain rules if there is no password. So I though it would be better to give the user control over the full url.